docs/override-configs.md

```diff
@@ -1,3 +1,29 @@
+## Enabling/disabling components
+
+To enable/disable components in Ozone, you need to modify the configurations in the [docker-compose-files.txt](https://github.com/ozone-his/ozone-docker-compose/blob/main/scripts/docker-compose-files.txt) in the ozone-docker-compose for your Ozone deployment. Here's how you can do it:
+
+**Access the excutable files**
```

Comment:
> excutable

This is not an executable file.
It's more of a configuration file, or a manifest.